src/Workspaces/Core/Portable/FindSymbols/SyntaxTree/SyntaxTreeIndex_Persistence.cs

```diff
@@ -16,7 +16,7 @@ namespace Microsoft.CodeAnalysis.FindSymbols
     internal sealed partial class SyntaxTreeIndex : IObjectWritable
     {
         private const string PersistenceName = "<SyntaxTreeIndex>";
-        private static readonly Checksum SerializationFormatChecksum = Checksum.Create("25");
+        private static readonly Checksum SerializationFormatChecksum = Checksum.Create("26");
```

Comment:
our binary format is the same, but we cannot use any existing caches as the new and old data are not compatible with each other.